app_dart/lib/src/service/scheduler.dart

```diff
@@ -297,6 +297,14 @@ class Scheduler {
     await luciBuildService.cancelBuilds(pullRequest, reason);
   }
 
+  Future<void> cancelPreSubmitTargetsV2({
+    required PullRequest pullRequest,
+    String reason = 'Newer commit available',
+  }) async {
+    log.info('Cancelling presubmit targets with buildbucket v2.');
+    await luciBuildService.cancelBuildsV2(pullRequest, reason);
+  }
```

Comment:
Ditto, do we still need this in v1?

Reply:
This one I do not think so but I would rather remove it in the follow up when we are no longer processing v1 events.